ecosystem/sep-0006.md

```diff
@@ -315,6 +317,7 @@ Using this feature, anchors will no longer have to wait until the user's Stellar
 
 **Anchors**: To support claimable balances anchors must
 
+* Set the `claimable_balances` attribute within `GET /info`'s `feature_flags` object to `true`
```

Comment:
Is there any negative affect on the situation where anchors have already implemented claimable balances but not the feature flags object?

I'm wondering if a client could be surprised, thinking that claimable balances would not be used because the default is false. 

Would it work to say that a missing features flag object means the client doesn't know what they support?

Reply:
Today, all wallets have to assume anchors do not support claimable balances, so I don't think this will be a problem.

If a wallet _knows_ an anchor supports claimable balances despite not yet updating their `GET /info` response, it can leave it's implementation unchanged -- passing `claimable_balance_supported: true` in the `GET /deposit` request and waiting for the claimable balance to be created. Or, it could implement the `features` detection while also adding an allowlist of sorts simultaneously. Its not perfect but I think the process is manageable.